### PR TITLE
Use calico 3.17 for k8s-1.17

### PR DESCRIPTION
**Pull request by qinqon**
Calico 3.18 [1] does not support k8s 1.17 anymore, this change change
k8s-1.17 provider calico version to the latest that support k8s 1.17 [2]

[1] https://github.com/projectcalico/calico/blob/v3.18.0/getting-started/kubernetes/requirements.md
[2] https://github.com/projectcalico/calico/blob/v3.17.3/getting-started/kubernetes/requirements.md

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: cluster-provision/k8s/1.17/manifests/cni.diff [hunk 1]
```diff
@@ -1,4 +1,5 @@
---- a/cluster-provision/k7s/1.17/manifests/cni.do-not-change.yaml
+index efabfe31..cc7d4658 100644
```
Comment (ormergi) [inline cluster-provision/k8s/1.17/manifests/cni.diff:1]: Do we need this line?

**Review by ormergi — COMMENTED** [review]
We dont need to support dual-stack for k8s-1.17, right?

File: cluster-provision/k8s/1.17/manifests/cni.diff (hunk 1 above)
Comment (qinqon) [reply cluster-provision/k8s/1.17/manifests/cni.diff]: Removed

**Comment by qinqon** [comment]
> We dont need to support dual-stack for k8s-1.17, right?

Nope

**Review by ormergi — APPROVED** [review]

**Review by fgimenez — APPROVED** [review]
/lgtm
/approve

**Pull request merged**